README.md

```diff
@@ -1,50 +1,11 @@
-# Account Abstraction SDK (aa-sdk)
+# aa-sdk
 
-The `aa-sdk` is a type-safe and performant TypeScript library built on top of [viem](https://viem.sh/) to provide ergonomic methods for sending user operations, sponsoring gas, and deploying smart contract accounts. It handles all the complexity of ERC-4337 under the hood to make account abstraction simple.
-
-The SDK also implements an EIP-1193 provider interface to easily plug into any popular dapp or wallet connect libraries such as RainbowKit, Wagmi, and Web3Modal. It also includes ethers.js adapters to provide full support for ethers.js apps.
-
-The `aa-sdk` is modular at every layer of the stack and can be easily extended to fit your custom needs. You can plug in any [smart account](https://accountkit.alchemy.com/smart-accounts/accounts/using-your-own) implementation, [Signer](https://accountkit.alchemy.com/smart-accounts/signers/overview), gas manager API, RPC provider.
+`aa-sdk` is a framework to embed smart accounts in your web3 app, unlocking powerful features like email/social login, gas sponsorship, batched transactions, and more. It makes it easy to integrate and deploy smart accounts, send user operations, and sponsor gas with just a few lines of code.
```

Comment:
you might've ripped this from Account Kit, but remember aa-sdk the SDK portion. Account Kit is the framework, but aa-sdk is the library. Let's write this closer to what was previously there.

Reply:
cool - lemme re-write but I still want this to be simple explanation instead of the explanation we had before since that's just too complicated to understand what this SDK is. 

Words like "EIP-1193 provider interface, ethers.js adapters etc."

Reply:
I really would rather we brought back the those things you mentioned. 

For example:
>The SDK also implements an EIP-1193 provider interface to easily plug into any popular dapp or wallet connect libraries such as RainbowKit, Wagmi, and Web3Modal. It also includes ethers.js adapters to provide full support for ethers.js apps.

This is an important bit of info! This is the readme in a github repo, a dev should be able to understand the value prop for them here and what they might find in the codebase. It answers the questions like 
"can I use this in my code base?" 
"Does it work with my other stuff?" 
"Wait it says viem, but I use ethers, I guess this isn't for me? Oh wait I can use it! nice."

I'd rather we keep most of what was here before and make sure we update links to link out to docs, but not cut content here

The updates to the Contributing section below are a solid simplification, so lets keep that

Reply:
Thanks for the explanation Moldy! Rolled it back!